Assets/Scripts/Models/Tile.cs

```diff
@@ -99,6 +96,7 @@ public Tile(int x, int y)
         this.X = x;
         this.Y = y;
         Characters = new List<Character>();
+        MovmentModifier = 1;
```

Comment:
It helps showing that it's a float if you set it to either `1f` or `1.0f`. I prefer the second, but both are good to indicate it's not an int.